### ISSUE TITLE
NVDA cannot read Greek documents with left and right arrow written in the jarte text editor

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2053]
**Reported by nikosdemetriou on 2012-01-19 20:26**
NVDA seams unable to read Greek documents using the left and right arrows with in the jarte text editor. It reads fine with up and down arrows or by word using control left and right arrows. Also when pressing nvda+up twice NVDA reads the letters ok so the problem is only with the left and right arrows on their own. When pressing left and right instead of hearing the letters NVDA reads different symbols such as micro, etc. I asked my sighted friends and they said that Greek looks ok on the screen so it doesn't seam to be something wrong with jarte itself but i might be wrong.

**Comment by bhavyashah** [comment]
While we may require the reported issue to be tested with a sample Greek PDF document in the latest version of Acrobat Reader, we probably cannot contact nikosdemetriou anymore. Thoughts? @ehollig @jcsteh

**Comment by Adriani90** [comment]
@nikosdemetriou are you still having this problem in NVDA 2019.1.1?

**Comment by nikosdemetriou** [comment]
Hi.
Thanks for contacting me.
I downloaded the latest version of jarte and it seams to work fine now.
I don't know when this issue was fixed exactly, because I contacted the jarte developer and he said that unicode languages were not supported in jarte and that he didn't have any plans of supporting them. 
After his response I forgot about jarte and I didn't use it since then.
But I am glad to see it working.
Nikos

**Comment by Adriani90** [comment]
Thanks, that sounds good. Closing as works for me.
